syntax = "proto3";

package security;

option java_multiple_files = true;
option java_package = "common.model.protos";
option java_outer_classname = "IdentifierTypeProtos";

enum IdentifierTypeProto {
  UNKNOWN_IDENTIFIER_TYPE=0;
  EXCH_TICKER=1;
  ISIN=2;
  CUSIP=3;
  OSI=4;
  FIGI=5;

  CASH=50;
}
